procedural/timeline/src/main/kotlin/gov/nasa/jpl/aerie/timeline/plan/AeriePostgresSimulationResults.kt

```diff
@@ -0,0 +1,167 @@
+package gov.nasa.jpl.aerie.timeline.plan
+
+import gov.nasa.jpl.aerie.merlin.driver.json.SerializedValueJsonParser
+import gov.nasa.jpl.aerie.merlin.protocol.types.SerializedValue
+import gov.nasa.jpl.aerie.merlin.protocol.types.Duration
+import gov.nasa.jpl.aerie.timeline.Interval.Companion.between
+import gov.nasa.jpl.aerie.timeline.Interval.Inclusivity.Exclusive
+import gov.nasa.jpl.aerie.timeline.Interval.Inclusivity.Inclusive
+import gov.nasa.jpl.aerie.timeline.collections.Instances
+import gov.nasa.jpl.aerie.timeline.ops.coalesce.CoalesceSegmentsOp
+import gov.nasa.jpl.aerie.timeline.payloads.Segment
+import gov.nasa.jpl.aerie.timeline.payloads.activities.Instance
+import java.io.StringReader
+import java.sql.Connection
+import javax.json.Json
+import kotlin.jvm.optionals.getOrNull
+
+/** A connection to Aerie's database for a particular simulation result. */
+data class AeriePostgresSimulationResults(
```

Comment:
My understanding is that we're moving the postgres capabilities out of the library itself - is that accurate?

Reply:
Yes. I've already moved some of the implementations out of the scheduling and timeline libraries into scheduler-driver, but not this one because I don't have a place to put it yet. I can make a procedural-utils library quickly.

Reply:
Added `...aerie.procedural.remote`.